data/analytics/attributes.yml

```diff
@@ -185,11 +225,35 @@
   type: string
   redact: false
 
+- name: navigation_list_type
+  description: Used in the page_view object to record the navigation list type
+  value: content attribute of meta tag govuk:navigation_list_type
+  example: curated
+  required: no
+  type: string
+  redact: false
+
+- name: navigation_page_type
+  description: Used in the page_view object to record the navigation page type
```

Comment:
Can we say any more about this? I'm not certain what the navigation page type is.

Reply:
Maybe ask a PA?

Reply:
@andysellick Have done this now and updated the description as a result, so should be ready to review again :+1:

Reply:
Nice work 👍